src/ks-app/src/app/containers/buttons/buttons.component.ts

```diff
@@ -6,10 +6,15 @@
 
 import {Component} from "@angular/core";
 import {Button, ClrButton, ClrLoadingButton, LoadingButton} from "@clr/angular";
+import {ClrLoadingState} from "@clr/angular";
```

Comment:
This should just map to the above import group right?

Reply:
yes. I'll update it.